deploy/stacks/codeartifact.py

```diff
@@ -36,6 +36,27 @@ def __init__(
             )
             domain_dict['permissions_policy_document'] = domain_policy
 
+            pip_repo_policy = iam.PolicyDocument(
+                statements=[
+                    iam.PolicyStatement(
+                        effect=iam.Effect.ALLOW,
+                        actions=[
+                            "codeartifact:DescribePackageVersion",
+                            "codeartifact:DescribeRepository",
+                            "codeartifact:GetPackageVersionReadme",
+                            "codeartifact:GetRepositoryEndpoint",
+                            "codeartifact:ListPackageVersionAssets",
+                            "codeartifact:ListPackageVersionDependencies",
+                            "codeartifact:ListPackageVersions",
+                            "codeartifact:ListPackages",
+                            "codeartifact:ReadFromRepository"
+                        ],
+                        principals=principals,
+                        resources=['*'],
+                    )
+                ]
+            )
+
```

Comment:
Just for my understanding. This block is needed for the cross-account permissions needed for installing packages using this repo from the DBMigration CodeBuild stage in the infra account right?

Reply:
Yes exactly - similar to the way we allow cross account access to the codeartifact domain we need to allow cross account access specifically to the PyPi Repo a part of the domain so that the DBMigration CodeBuild can install packages directly from there